ddtrace/internal/compat.py

```diff
@@ -117,6 +117,16 @@ def process_time_ns():
         return int(_process_time() * 1e9)
 
 
+if sys.version_info < (3, 8):
```

Comment:
To support TypedDicts in python versions less than 3.8 I will need to add `typing-extensions` to ddtrace-py/setup.py. Is this okay?

Reply:
Minimum version of typing extension we need is [3.7.4](https://pypi.org/project/typing-extensions/3.7.4/).

Reply:
I removed typing extensions